Automattic/studio
# Studio: Update WordPress.com logo

Fixes #620 

## Proposed Changes

- Update the WordPress.com logo to the latest version.
- Update spacing and alignment in the header bar for new logo.

| Before | After |
|--------|--------|
| <img width="316" alt="Screenshot 2024-10-24 at 14 24 32" src="https://github.com/user-attachments/assets/d3128143-e5f4-4ed8-b537-27b170d6ddee"> | <img width="316" alt="Screenshot 2024-10-24 at 14 24 22" src="https://github.com/user-attachments/assets/fdf573d2-abed-473b-9421-16ca8b418179"> | 

## Testing Instructions



- Open Studio
- Ensure you are logged out of WordPress.com

## Pre-merge Checklist
